platform/ui/src/components/index.js

```diff
@@ -79,10 +79,25 @@ import PanelSection from './PanelSection';
 import AdvancedToolbox from './AdvancedToolbox';
 import InputDoubleRange from './InputDoubleRange';
 import LegacyButtonGroup from './LegacyButtonGroup';
+import SwitchButton, { SwitchLabelLocation } from './SwitchButton';
+import {
+  AllInOneMenu,
+  AllInOneMenuDividerItem,
+  AllInOneMenuHeaderItem,
+  AllInOneMenuItem,
+  AllInOneMenuItemPanel,
+  AllInOneMenuSubMenu,
+} from './AllInOneMenu';
```

Comment:
two things

1. can we namespace this?

so `import * as AllInOneMenu` so that we can use `AllInOneMenu.AllInOneMenuDividerItem` 

2. The AllInOneMenu might be redundant and too noisy, so how about we rename things so that we can do `AllInOneMenu.DividerItem|HeaderItem|Item|ItemPanel`